### DISCUSSION TITLE
1.16.5 version can no longer build

### DISCUSSION THREAD

**Discussion by SonyStinks** [P3pp3rF1y/SophisticatedBackpacks#1036] [General]
It seems the "me.sizableshrimp.parchmenttest" dependency no longer exists where it once did. I've checked that maven and it's still there, but all the file links give 404 errors...
I tried updating the build settings to use regular Parchment from parchmentmc.org, added the new ForgeGradle version, and gave up when Gradle could not find a compile() method for vaskii's AutoRegLib. At that point the build.gradle looked pretty similar to the one you use in the 1.18+ branches.
I have no idea how, or even IF, you'd like to handle this problem. Especially given I'm only building the .jar file "because that's the fun of open source!"...but in case it's an _actual_ problem. I thought I should tell you about it.

Cheers!

**Comment by P3pp3rF1y** [comment]
I don't support 1.16 for a very long time now so you're really on your own to figure these issues out, autoreglib just sounds like you can find a different version that's still available for 1.16 or possibly use it from different repo / directly from curseforge (don't remember if I did back then). And parchment I believe has changed where it's hosted since that parchmenttest thing, but if you can't get this working you can just get rid of it altogether, it only provides sensible names for parameters so in the worst case you will have to deal with seeing p_##### parameters in vanilla code but that doesn't affect SB code.

**Comment by SonyStinks** [comment]
Thanks for the info. SB is really an awesome mod and one of my go-to mods for any pack I host, so eventually I'll manage to figure out all the out-of-dates & updates and get a working build for package testing, at which point I'll try contacting you again about merging a PR.  How's that sound?

**Comment by P3pp3rF1y** [comment]
What is going to be in that PR? At this point I would only do critical fixes but I can't imagine that there would be any given that forge hasn't updated in years and I haven't heard of any issues in those years

**Comment by SonyStinks** [comment]
Sorry it took me a while to get back to you. The PR I mentioned would only contain whatever changes are necessary for the package to build normally again. So mostly updates to out-of-date links and some syntax changes to things in the Gradle files that IntelliJ IDEA doesn't understand how to handle.  The PR is not necessary by any means, but some people like to know what's been done with their code and I figured having the old package build properly would be a nice thing to have. No critical fixes and I don't plan on adding any new features!
